src/figuresClasses.ts

```diff
@@ -1,19 +1,106 @@
-export interface Figure {
 
+export interface Figure {
+  shape: string;
+  color: string;
+  getArea(): number;
 }
 
-export class Triangle {
+export class Triangle implements Figure {
+  shape: string;
 
-}
+  color: string;
+
+  a: number;
+
+  b: number;
+
+  c: number;
 
-export class Circle {
+  private s: number;
 
+  constructor(color: string, a: number, b: number, c: number) {
+    this.shape = 'triangle';
```

Comment:
Let's create the enum `Shape` with the values so that it could restrict the values that passed to the shape property